src/Agent/NewRelic/Agent/Core/Agent.cs

```diff
@@ -292,60 +293,25 @@ public void HandleWrapperException(Exception exception)
 
         public Stream TryGetStreamInjector(Stream stream, Encoding encoding, string contentType, string requestPath)
         {
-            if (stream == null)
-            {
-                return null;
-            }
-
-            if (encoding == null)
-            {
-                return null;
-            }
-
-            if (contentType == null)
-            {
-                return null;
-            }
-
-            if (requestPath == null)
-            {
-                return null;
-            }
-
-            try
-            {
-                var transaction = _transactionService.GetCurrentInternalTransaction();
-                if (transaction == null)
-                {
-                    return null;
-                }
-
-                var shouldInject = _browserMonitoringPrereqChecker.ShouldAutomaticallyInject(transaction, requestPath, contentType);
-                if (!shouldInject)
-                {
-                    return null;
-                }
+            var script = TryGetRUMScriptInternal(contentType, requestPath);
```

Comment:
To maintain the existing behavior, we may want to keep the stream and encoding checks in this method.

Reply:
Good catch.